csrc/aio/py_test/parse_aio_stats.py

```diff
@@ -9,31 +9,12 @@
 import argparse
 import re
 
-RAW_RATE = 'raw_rate'
-E2E_RATE = 'e2e_rate'
-SUBMIT_LATENCY = 'submit_latency'
-COMPLETE_LATENCY = 'complete_latency'
 READ_SPEED = 'read_speed'
 WRITE_SPEED = 'write_speed'
 
-TASK_READ_SPEED = 'task_read_speed'
-
-PERF_METRICS = [
-    RAW_RATE,
-    E2E_RATE,
-    SUBMIT_LATENCY,
-    COMPLETE_LATENCY,
-    READ_SPEED,
-    WRITE_SPEED
-]
-METRIC_SEARCH = {
-    RAW_RATE: 'ds_raw_time',
-    E2E_RATE: 'ds_time',
-    SUBMIT_LATENCY: 'aggr: submit',
-    COMPLETE_LATENCY: 'aggr: complete',
-    READ_SPEED: 'E2E Read Speed',
-    WRITE_SPEED: 'E2E Write Speed'
-}
+PERF_METRICS = [READ_SPEED, WRITE_SPEED]
+
+METRIC_SEARCH = {READ_SPEED: 'E2E Read Speed', WRITE_SPEED: 'E2E Write Speed'}
 
 NUM_BYTES = (400 * 1024 * 1024)
 NUM_GIGA_BYTES = (1024 * 1024 * 1024)

```

Comment:
Could we rename to something like `BYTES_PER_GB` or just `GIGA` ? I think `NUM_GIGA_BYTES` might be misleading since its used to compute the number of GB.

Reply:
Turns out this is dead code anyways :)